src/analyserequationast.cpp

```diff
@@ -18,6 +18,10 @@ limitations under the License.
 
 #include "analyserequationast_p.h"
 
+#ifdef NAN
+#    undef NAN
+#endif
+
```

Comment:
Hasn't this already been done elsewhere? Why the repatiion

Reply:
As a result of some cleaning up (see 57289480).